stdlib/public/TensorFlow/TensorGroup.swift

```diff
@@ -49,3 +51,82 @@ public extension TensorGroup {
     return Array(repeating: nil, count: Int(_tensorHandleCount))
   }
 }
+
+//===----------------------------------------------------------------------===//
+// Conform standard TensorFlow types to TensorGroup
+//===----------------------------------------------------------------------===//
+
+extension TensorHandle : TensorGroup where Scalar : AccelerableByTensorFlow {
+  public static var _typeList: [TensorDataType] {
+    return [Scalar.tensorFlowDataType]
+  }
+
+  public func _unpackTensorHandles(
+      into address: UnsafeMutablePointer<CTensorHandle>?) {
```

Comment:
do we always force unwrap `address` via `!`? if so, should we pass in a non-optional `UnsafeMutablePointer<CTensorHandle>` instead?

Reply:
`_TFCOpAddInputFromTensorGroup` passes it the `baseAddress` of an `UnsafeMutableBufferPointer`, which is an optional that is allowed to be nil when the buffer has 0 elements, which can happen for 0-element structs.

Reply:
thanks. please document this case in https://github.com/apple/swift/blob/a4a9704b6019e44be7e0b4a0aa439a67b86f8561/stdlib/public/TensorFlow/TensorGroup.swift#L35. Same for `init()`?

Reply:
Also, is there really such a thing as a 0-element struct?

Reply:
Yes, there is.